windows/src/Makefile

```diff
@@ -95,11 +96,9 @@ unit-tests:
 release: clean-pathdefines test-release-cef-path test-releasebuildcheck test-uiaccess test-klog test-releaseexists test-cert clean
     cd $(ROOT)\src
     $(MAKE) global-versions
-    $(MAKE) disable-search
     $(MAKE) build-release-wrapper
-    $(MAKE) clean    
+    # $(MAKE) clean  <-- this makes it harder to do OEM builds
```

Comment:
So, do we do a `make clean` before all builds manually first?  Or is there no `clean` at all as part of the process now?

Reply:
`make clean` is really a legacy thing when doing release builds. It was there when we were running release builds locally. Now that they are done on the build agents where they start clean anyway, it's not necessary and removing the clean step makes it easier to integrate the FirstVoices build which has dependencies on the build.